internal/graph/check.go

```diff
@@ -627,15 +626,6 @@ func (c *LocalChecker) checkDirect(parentctx context.Context, req *ResolveCheckR
 					return nil, err
 				}
 
-				if len(condEvalResult.MissingParameters) > 0 {
```

Comment:
If I may be able to pitch in, in my opinion we should take a different approach and ensure that we do not support partial application of conditions params and instead always require all the params of a given condition and error if some are not present. This to me feels like a more straightforward approach that sets better expectations going forward.